src/Stripe.net/Services/StripeRequestOptions.cs

```diff
@@ -5,5 +5,8 @@ public class StripeRequestOptions
         public string ApiKey { get; set; }
         public string StripeConnectAccountId { get; set; }
         public string IdempotencyKey { get; set; }
+        // This is used specifically for Ephemeral Keys as they have to be created
+        // with a specific API version. Don't use it for anything else.
+        internal string StripeVersion { get; set; }
```

Comment:
Thanks Remi. That looks good to me!

Reply:
@remi-stripe @brandur-stripe  So when this pull request will be available  , I can't stand to use it , my ios mobile developer need this feature to complete the job .

Reply:
Shouldn't this be public? Right now, user is unable to specify the `StripeVersion` when requesting the ephemeral key as this property is `internal` to the library (and tests - `InternalsVisibleTo`).

Reply:
@jakubfijalkowski ha nice catch, I assumed the tests would have failed similarly to how a code using the library would have failed. I'll have to figure out a better solution than this to set the property then.
@brandur-stripe do you have an opinion here?

Reply:
Okay I think I found the right fix and pushed it in https://github.com/stripe/stripe-dotnet/pull/997/commits/8490d0f4e5b8a192c102a119680830f80c178474

Reply:
@remi-stripe Great! I've left some other small comments, feel free to ignore them. ;)